packages/aws-lambda/lambdas/runtime_emulator/Dockerfile

```diff
@@ -12,7 +12,11 @@ COPY --from=instana-layer /opt/extensions/ /opt/extensions/
 COPY --from=instana-layer /opt/nodejs/ /opt/nodejs/
 
 # Copy function code
+# COPY esm/app.js ${LAMBDA_TASK_ROOT}
+# COPY esm/package.json ${LAMBDA_TASK_ROOT}
```

Comment:
_nitpick:_ Do we need these two commented out lines for testing esm? Do you want to add a comment here as well to explain that?

Reply:
Yes if the developer wants to test esm, he also needs to modify the Dockerfile. 
I though the comment in line 19/20 is enough? What do you think?

Reply:
sure